src/interactive/AdminBar.jsx

```diff
@@ -0,0 +1,239 @@
+import Section from '../layout/Section';
+import Tooltip from './Tooltip';
+import SelectInput from '../forms/SelectInput';
+
+type DropdownProps = {
+	group: Group,
+	event?: EventInfo,
+	host: string,
+};
+
+type Props = {
+	group: Group,
+	event?: EventInfo,
+	isQL: boolean,
+	isAdmin: boolean,
+	isProd: boolean,
+	self: Self,
+	nodeEnv: string,
+};
+
+type State = {
+	highlightValue: string,
+	showHighlighter: boolean,
+};
+
+const DropdownContent = ({ group, event, host }: DropdownProps): React$Node => (
+	<ul className="padding--all">
+		{group.organizer && (
+			<li>
+				<a
+					href={`https://www.${host}/ql/?qlMemberId=${group.organizer.id}`}
+					className="link"
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					QL Org
+				</a>
+			</li>
+		)}
+		<li>
+			<a
+				href={`https://admin.${host}/admin/group.jsp?g=${group.id}&op=search`}
+				className="link"
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				Group
+			</a>
+		</li>
+		{group.organizer && (
+			<li>
+				<a
+					href={`https://admin.${host}/admin/member.jsp?m=${
+						group.organizer.id
+					}`}
+					className="link"
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					Org
+				</a>
+			</li>
+		)}
+		{event !== undefined &&
+			event.id && (
+				<li>
+					<a
+						href={`https://admin.${host}/admin/events.jsp?eventIdOrUrl=${
+							event.id
+						}`}
+						className="link"
+						target="_blank"
+						rel="noopener noreferrer"
+					>
+						Event
+					</a>
+				</li>
+			)}
+		<li>
+			<a
+				href={`https://admin.${host}/admin/events.jsp?groupIdOrUrl=${group.id}`}
+				className="link"
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				Group Events
+			</a>
+		</li>
+		<li>
+			<a
+				href={`https://admin.${host}/admin/notifications/chapter/${group.id}`}
+				className="link"
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				Notifs
+			</a>
+		</li>
+	</ul>
+);
+
+export class AdminBar extends React.PureComponent<Props, State> {
+	constructor(props: Props) {
+		super(props);
+		this.state = {
+			highlightValue: '1',
+			showHighlighter: false,
+		};
+	}
+
+	onHighlightValueChange = (e: SyntheticInputEvent<EventTarget>) => {
+		this.setState({ highlightValue: e.target.value });
+	};
+
+	highlightGroup = (host: string) => {
+		fetch(`https://admin.${host}/admin_api/index/highlight/`, {
+			method: 'POST',
+			headers: { 'content-type': 'application/x-www-form-urlencoded' },
+			body: `chapter_id=${this.props.group.id}&value=${this.state.highlightValue}`,
+			credentials: 'include',
+		}).then(() => {
+			this.props.group.highlight = this.state.highlightValue;
+			this.toggleHighlighter();
+		});
+	};
+
+	toggleHighlighter = () => {
+		this.setState(state => ({ showHighlighter: !state.showHighlighter }));
+	};
+
+	render() {
+		const { group, event, isQL, isAdmin, self, isProd, nodeEnv } = this.props;
+		const host = nodeEnv === 'production' ? 'meetup.com' : 'dev.meetup.com';
```

Comment:
question: should this be using `nodeEnv` or `isProd` to determine the host? i don't know and am curious

Reply:
In case when we are working on dev env, but using production data it is : `isProd: true, nodeEnv: 'development', host: `dev.meetup.com'`

Reply:
I think that when working with prod data, you probably want to link to prod admin, so you could use `isProd` (although I would recommend renaming to `isProdApi` to remove ambiguity with `process.env.NODE_ENV`).